extensions/amp-animation/0.1/web-animations.js

```diff
@@ -1225,12 +1242,13 @@ class CssContextImpl {
     let element;
     try {
       if (selectionMethod == 'closest') {
-        element = closestAncestorElementBySelector(
+        element = closest(
```

Comment:
Hmm. This is so much slower though. I'd much rather we do the actual `found = closestAncestorElementBySelector()` and simply check if `this.scope_.contains(found) ? found : null`.